pkgs/games/minecraft/default.nix

```diff
@@ -49,6 +45,10 @@ in stdenv.mkDerivation {
 
     mkdir -p $out/share/applications
     ln -s ${desktopItem}/share/applications/* $out/share/applications/
+
+    ${openjdk}/bin/jar xf $out/minecraft.jar favicon.png
+    mkdir -p $out/share/icons/hicolor/32x32/apps/
+    cp favicon.png $out/share/icons/hicolor/32x32/apps/minecraft.png
   '';
```

Comment:
I've recently discovered a nice command `install`. It does `mkdir -p && cp`
```
install -D $out/share/icons/hicolor/32x32/apps/minecraft.png favicon.png
```
Nice trick, when you want to remove double `$out/share/icons/hicolor/32x32/apps/`

This is just FYI, not a critique of this PR